docs/content/specs/shared/_index.md

```diff
@@ -506,22 +506,34 @@ Unless explicitly requested and agreed, members of the AVM core team or any PG t
 
-| GitHub Team Name                                     | Description                                                                  | Permissions | Permissions granted through                                        | Where to work?          |
-|------------------------------------------------------|------------------------------------------------------------------------------|-------------|--------------------------------------------------------------------|-------------------------|
-| `<hyphenated module name>-module-owners-bicep`       | Module Owners of the <module name> AVM Bicep <resource/pattern> module       | **Write**   | Assignment to the **`avm-technical-reviewers-bicep`** parent team. | Need to work in a fork. |
-| `<hyphenated module name>-module-contributors-bicep` | Module Contributors of the <module name> AVM Bicep <resource/pattern> module | **Triage**  | **`avm-module-contributors-bicep`** parent team.                   | Need to work in a fork. |
+| GitHub Team Name                                     | Description                                    | Permissions | Permissions granted through                                        | Where to work?          |
+|------------------------------------------------------|------------------------------------------------|-------------|--------------------------------------------------------------------|-------------------------|
+| `<hyphenated module name>-module-owners-bicep`       | AVM Bicep Module Owners - \<module name>       | **Write**   | Assignment to the **`avm-technical-reviewers-bicep`** parent team. | Need to work in a fork. |
+| `<hyphenated module name>-module-contributors-bicep` | AVM Bicep Module Contributors - \<module name> | **Triage**  | **`avm-module-contributors-bicep`** parent team.                   | Need to work in a fork. |
 
 Examples - GitHub teams required for the Bicep resource module of Azure Virtual Network (`avm/res/network/virtual-network`):
 
 - `avm-res-network-virtualnetwork-module-owners-bicep` --> assign to the `avm-technical-reviewers-bicep` parent team.
 - `avm-res-network-virtualnetwork-module-contributors-bicep` --> assign to the `avm-module-contributors-bicep` parent team.
 
+{{< hint type=tip >}}
+Direct link to create a new GitHub team and assign it to its parent: [Create new team](https://github.com/orgs/Azure/new-team)
+
+Fill in the values as follows:
+
+- **Team name**: Following the naming convention described above, use the value defined in the module indexes.
+- **Description**: Follow the guidance above (see the Description column in the table above).
+- **Parent team**: Follow the guidance above (see the Permissions granted through column in the table above).
+- **Team visibility**: `Visible`
+- **Team notifications**: `Enabled`
+{{< /hint >}}
```

Comment:
What about a screenshot here? If so with usual style, dark mode with light green boxes to highlight content

Reply:
I would avoid that. There's already a counter argument of some of the requirements being too lengthy. This one is definitely one of them. Adding a screenshot wouldn't add any value, as the UI is very simple, it only consists of these points listed. Also, adding a screenshot would need to be in addition to the text explaining (and not instead of), as the guidance is different for Bicep vs. TF. Hence, I would really not like to put one here. If the feedback is that this still needs further explanation, it should be part of a future module owners' guide.

Reply:
Fair enough, I'll approve since this is actually specs and I agree it's already too long as it is. Some of this content should probably be moved to the contribution guidelines for module owners instead. Let's rediscuss screenshots at that time